astronomer/providers/amazon/aws/example_dags/example_emr_eks_containers_job.py

```diff
@@ -0,0 +1,156 @@
+        "sparkSubmitParameters": "--conf spark.executors.instances=2 --conf spark.executors.memory=2G --conf spark.executor.cores=2 --conf spark.driver.cores=1",  # noqa: E501
+    }
+}
+
+CONFIGURATION_OVERRIDES_ARG = {
+    "applicationConfiguration": [
+        {
+            "classification": "spark-defaults",
+            "properties": {
+                "spark.hadoop.hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory",  # noqa: E501
+            },
+        }
+    ],
+    "monitoringConfiguration": {
+        "cloudWatchMonitoringConfiguration": {
+            "logGroupName": "/aws/emr-eks-spark",
+            "logStreamNamePrefix": "airflow",
+        }
+    },
+}
+# [END howto_operator_emr_eks_config]
+
+with DAG(
+    dag_id="emr_eks_pi_job",
+    start_date=datetime(2022, 1, 1),
+    schedule_interval=None,
+    catchup=False,
+    default_args=default_args,
+    tags=["emr_containers", "example"],
+) as dag:
+    # Task steps for DAG to be self-sufficient
+    setup_aws_config = BashOperator(
+        task_id="setup_aws_config",
+        bash_command=f"aws configure set aws_access_key_id {AWS_ACCESS_KEY_ID}; "
+        f"aws configure set aws_secret_access_key {AWS_SECRET_ACCESS_KEY}; "
+        f"aws configure set default.region {AWS_DEFAULT_REGION}; ",
+    )
+
+    # Task to create EMR clusters on EKS
+    create_EKS_cluster_kube_namespace_with_role = BashOperator(
+        task_id="create_EKS_cluster_kube_namespace_with_role",
+        bash_command="sh $AIRFLOW_HOME/dags/example_create_emr_on_eks_cluster.sh ",
+    )
+
+    # Task to create EMR virtual cluster
+    create_EMR_virtual_cluster = PythonOperator(
+        task_id="create_EMR_virtual_cluster",
+        python_callable=create_emr_virtual_cluster_func,
+    )
+
+    # An example of how to get the cluster id and arn from an Airflow connection
+    # VIRTUAL_CLUSTER_ID = '{{ conn.emr_eks.extra_dejson["virtual_cluster_id"] }}'
+    # JOB_ROLE_ARN = '{{ conn.emr_eks.extra_dejson["job_role_arn"] }}'
+
+    # [START howto_operator_emr_eks_jobrun]
+    job_starter = EmrContainerOperator(
+        task_id="start_job",
+        virtual_cluster_id=VIRTUAL_CLUSTER_ID,
+        execution_role_arn=JOB_ROLE_ARN,
+        release_label="emr-6.2.0-latest",
+        job_driver=JOB_DRIVER_ARG,
+        configuration_overrides=CONFIGURATION_OVERRIDES_ARG,
+        name="pi.py",
+    )
+    # [END howto_operator_emr_eks_jobrun]
+
+    # [START howto_sensor_emr_container_task]
+    job_container_sensor = EmrContainerSensorAsync(
+        task_id="check_container_sensor",
+        job_id=job_starter.output,
+        virtual_cluster_id=VIRTUAL_CLUSTER_ID,
+        poll_interval=5,
+        aws_conn_id=AWS_CONN_ID,
+    )
+    # [END howto_sensor_emr_container_task]
+
+    # Delete clusters, container providers, role, policy
+    remove_clusters_role_policy = BashOperator(
+        task_id="last_step_clean_up_cluster_role_policy",
```

Comment:
```suggestion
        task_id="clean_up_cluster_role_policy",
```

Reply:
Fixed to `remove_clusters_container_role_policy`